### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by TDGNate**
Added starter code for each route, linking up early to frontend so there isn't as many bugs as last time when we linked them all up last minute. Each route links up to their corresponding handlebar page.

### CODE REVIEW

File: controllers/index.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 const homeRoutes = require('./homeRoutes');
+const adminRoutes = require('./adminRoutes');
+const eventRoutes = require('./eventRoutes');
+const galleryRoutes = require('./galleryRoutes');
+const contactRoutes = require('./contactRoutes');
+const aboutRoutes = require('./aboutRoutes');
 
 router.use('/', homeRoutes);
+router.use('/admin', adminRoutes);
+router.use('events', eventRoutes);
```
Comment (TonyQ032) [inline controllers/index.js]: Should be `'/events'`. Route is currently broken and returns a 404 error when it should be a 500. Just put the / and it should fix it :)

**Review by TonyQ032 — CHANGES_REQUESTED** [review]
Overall fantastic job! Just need to make that one small change and its ready to merge into main!

File: controllers/index.js (hunk 1 above)
Comment (TDGNate) [reply controllers/index.js]: That is so cool that you can highlight a certain line or lines and add a comment 😂 
Thanks Tony, it's fixed now! 👍

**Review by TonyQ032 — APPROVED** [review]
Fantastic work! Approved!

**Pull request merged**